This adds #include <cstdint> to deal with 'uint64_t' does not name a type problem. 
Taken from here: https://github.com/AirenSoft/OvenMediaEngine/issues/1271
Author: J. Sassmannshausen (Imperial College London/UK) 
diff --git a/DRAGMAP-1.3.0.orig/src/include/sequences/Read.hpp b/DRAGMAP-1.3.0/src/include/sequences/Read.hpp
index 460c1cb..8d20cd6 100644
--- a/DRAGMAP-1.3.0.orig/src/include/sequences/Read.hpp
+++ b/DRAGMAP-1.3.0/src/include/sequences/Read.hpp
@@ -18,6 +18,7 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 namespace dragenos {
 namespace sequences {
diff --git a/DRAGMAP-1.3.0.orig/src/lib/sequences/tests/unit/CrcHasherMocks.hpp b/DRAGMAP-1.3.0/src/lib/sequences/tests/unit/CrcHasherMocks.hpp
index 1866be7..ca495cc 100644
--- a/DRAGMAP-1.3.0.orig/src/lib/sequences/tests/unit/CrcHasherMocks.hpp
+++ b/DRAGMAP-1.3.0/src/lib/sequences/tests/unit/CrcHasherMocks.hpp
@@ -3,6 +3,7 @@
 #include <array>
 #include <cassert>
 #include <cstdlib>
+#include <cstdint>
 #include <string>
 
 namespace dragenos {
diff --git a/DRAGMAP-1.3.0.orig/stubs/dragen/src/host/dragen_api/read_group_list.hpp b/DRAGMAP-1.3.0/stubs/dragen/src/host/dragen_api/read_group_list.hpp
index eefb9ae..623a77f 100644
--- a/DRAGMAP-1.3.0.orig/stubs/dragen/src/host/dragen_api/read_group_list.hpp
+++ b/DRAGMAP-1.3.0/stubs/dragen/src/host/dragen_api/read_group_list.hpp
@@ -14,6 +14,7 @@
 #define __READ_GROUP_LIST_HPP__
 
 #include "dragen_exception.hpp"
+#include <cstdint>
 class ReadGroupList {
 public:
   const std::string &getReadGroupName(const uint16_t idx) const {
